packages/site/src/guidesList.ts

```diff
@@ -9,7 +9,7 @@ export const guidesList = [
     additionalMatches: ["Custom", "Composability"],
   },
   {
-    title: "Documentation styleguide",
+    title: "Writing documentation",
```

Comment:
A slightly shorter name to wrap less, and also feels more of a piece with

Customizing components

Getting started with React

etc